perfkitbenchmarker/openstack/os_virtual_machine.py

```diff
@@ -34,6 +35,11 @@
 flags.DEFINE_integer('openstack_volume_size', 20,
                      'Size of the volume (GB)')
 
+flags.DEFINE_enum('openstack_scheduler_policy', NONE,
```

Comment:
Afinity and ant-Afinity are conflicting options then i think just one option is to use it as a switch.